# yellow
yellow:
  h_max: 40
  h_min: 20
  s_max: 255
  s_min: 35                    
  v_max: 255
  v_min: 100

#red
red:
  h_max: 20
  h_min: 0
  s_max: 255
  s_min: 241
  v_max: 94
  v_min: 133

#blue
blue:
  h_max: 80
  h_min: 60
  s_max: 255
  s_min: 35
  v_max: 255
  v_min: 100

# filter params
gaussian:
  size: 5
  flag: true

bilateral:
  ksize: 3
  sigma: 100
  flag: false

morph:
  open_ksize: 3
  close_ksize: 5
  flag: true
